packages/create-rule/src/handlebars-utils.ts

```diff
@@ -0,0 +1,46 @@
+import * as Handlebars from 'handlebars';
```

Comment:
This utils will be shared with `create-parser` it should be in a different place